bench-tps/src/cli.rs

```diff
@@ -355,6 +359,21 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
                 .validator(|arg| is_within_range(arg, 1, usize::MAX - 1))
                 .help("The number of unique destination accounts per transactions 'chunk'. Lower values will result in more transaction conflicts.")
         )
+        .arg(
+            Arg::with_name("bind_address")
+                .long("bind-address")
+                .value_name("HOST")
+                .takes_value(true)
+                .validator(solana_net_utils::is_host)
+                .help("IP address to use with connection cache"),
+        )
+        .arg(
+            Arg::with_name("client_node_id")
+                .long("client-node-id")
```

Comment:
Can we use "identity" CLI argument instead of adding a new one?

Reply:
This `identity` is mislead name, I would rename it to something like `funding-authority` because it is currently used to fund created accounts: https://github.com/solana-labs/solana/blob/master/bench-tps/src/bench.rs#LL996C13-L996C13

But I will have to use different name for the current node_id because if I will use identity and rename identity to something else, this could be misleading for existing users of this argument if any

Reply:
Ok, that makes sense.